powerline/segments/i3wm.py

```diff
@@ -16,12 +16,12 @@ def calcgrp(w):
 	return group
 
 
-def workspaces(pl):
+def workspaces(pl, strip=0):
 	'''Return workspace list
 
 	Highlight groups used: ``workspace``, ``w_visible``, ``w_focused``, ``w_urgent``
 	'''
 	return [{
-		'contents': w['name'],
+		'contents': w['name'][min(len(w['name']),strip):],
```

Comment:
I am wondering what for this segment exists in first place. If you can switch to workspaces by number then they should not be kept as long as possible, even if this means stripping names. If you can switch to workspaces by their names then they have priority. I do not know i3, but usually you do switch to workspaces according to their numbers.

There are also no examples of regex-based stripping.